diff --git a/node_modules/@aws-sdk/client-route-53/dist/cjs/protocols/Aws_restXml.js b/node_modules/@aws-sdk/client-route-53/dist/cjs/protocols/Aws_restXml.js
index 9a8089a..752ee5f 100644
--- a/node_modules/@aws-sdk/client-route-53/dist/cjs/protocols/Aws_restXml.js
+++ b/node_modules/@aws-sdk/client-route-53/dist/cjs/protocols/Aws_restXml.js
@@ -6992,8 +6992,8 @@ const deserializeAws_restXmlInvalidChangeBatchResponse = async (parsedOutput, co
         messages: undefined,
     };
     const data = parsedOutput.body.Error;
-    if (data["message"] !== undefined) {
-        contents.message = data["message"];
+    if (data["Message"] !== undefined) {
+        contents.message = data["Message"];
     }
     if (data.messages === "") {
         contents.messages = [];
@@ -7024,8 +7024,8 @@ const deserializeAws_restXmlInvalidInputResponse = async (parsedOutput, context)
         message: undefined,
     };
     const data = parsedOutput.body.Error;
-    if (data["message"] !== undefined) {
-        contents.message = data["message"];
+    if (data["Message"] !== undefined) {
+        contents.message = data["Message"];
     }
     return contents;
 };
@@ -7258,8 +7258,8 @@ const deserializeAws_restXmlNoSuchHealthCheckResponse = async (parsedOutput, con
         message: undefined,
     };
     const data = parsedOutput.body.Error;
-    if (data["message"] !== undefined) {
-        contents.message = data["message"];
+    if (data["Message"] !== undefined) {
+        contents.message = data["Message"];
     }
     return contents;
 };
@@ -7271,8 +7271,8 @@ const deserializeAws_restXmlNoSuchHostedZoneResponse = async (parsedOutput, cont
         message: undefined,
     };
     const data = parsedOutput.body.Error;
-    if (data["message"] !== undefined) {
-        contents.message = data["message"];
+    if (data["Message"] !== undefined) {
+        contents.message = data["Message"];
     }
     return contents;
 };
@@ -7349,8 +7349,8 @@ const deserializeAws_restXmlPriorRequestNotCompleteResponse = async (parsedOutpu
         message: undefined,
     };
     const data = parsedOutput.body.Error;
-    if (data["message"] !== undefined) {
-        contents.message = data["message"];
+    if (data["Message"] !== undefined) {
+        contents.message = data["Message"];
     }
     return contents;
 };
