diff --git a/node_modules/@kubernetes/client-node/dist/gen/model/networkingV1beta1IngressBackend.d.ts b/node_modules/@kubernetes/client-node/dist/gen/model/networkingV1beta1IngressBackend.d.ts
index 3c0df28..d673861 100644
--- a/node_modules/@kubernetes/client-node/dist/gen/model/networkingV1beta1IngressBackend.d.ts
+++ b/node_modules/@kubernetes/client-node/dist/gen/model/networkingV1beta1IngressBackend.d.ts
@@ -22,7 +22,7 @@ export declare class NetworkingV1beta1IngressBackend {
     /**
     * Specifies the port of the referenced service.
     */
-    'servicePort'?: object;
+    'servicePort'?: string | number;
     static discriminator: string | undefined;
     static attributeTypeMap: Array<{
         name: string;
